lib/cosmos/tools/table_manager/table_manager.rb

```diff
@@ -273,17 +337,23 @@ def initialize_actions(no_tables = false)
         @table_commit.statusTip = tr('Incorporate the current table data into a binary file which already contains the table')
         @table_commit.connect(SIGNAL('triggered()')) { table_commit() }
 
-        @table_update = Qt::Action.new(tr('&Update Definition'), self)
-        @table_update.statusTip = tr('Change the defaults in the definition file to the displayed table data')
-        @table_update.connect(SIGNAL('triggered()')) { table_update() }
+#        @table_update = Qt::Action.new(tr('&Update Definition'), self)
+#        @table_update.statusTip = tr('Change the defaults in the definition file to the displayed table data')
+#        @table_update.connect(SIGNAL('triggered()')) { table_update() }
```

Comment:
I don't think this was really used and automating definition updates seems pretty dangerous so I eliminated it.